qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -252,6 +252,11 @@ class AerSimulator(AerBackend):
       number of active circuit qubits is greater than this value batching of
       simulation shots will not be used. (Default: 16).
 
+    * ``num_threads_per_group`` (int): This option sets the number of
+      threads per group. For GPU simulation, this value sets number of
+      threads per GPU. This parameter is used to optimize Pauli noise
+      simulation with multiple-GPUs (Default: 1).
+
```

Comment:
`group` sounds too general. Can we rename this with `num_threads_per_device`?

Reply:
changed to num_threads_per_device